### ISSUE TITLE
how to transfer the sound file to *.wav file?

### ISSUE THREAD

**Issue by skyfall1970** [alexa/avs-device-sdk#397]
**IMPORTANT**: Before you create an issue, please take a look at our [Issue Reporting Guide](https://github.com/alexa/avs-device-sdk/wiki/Issue-Reporting-Guide).

## Briefly summarize your issue:

in source code, how to modified that code to play *.wav sound?
## What is the expected behavior?

play *wav
### What version of the AVS Device SDK are you using?
AVS Device SDK  1.2.0
ubuntu 14.04  64 bit

### Tell us what hardware you're using:
- [v ] Desktop / Laptop
- [ ] Raspberry Pi
- [ ] Other - tell us more:

### Tell us about your OS (Type & version):
- [ v] Linux
- [ ] MacOS
- [ ] Raspbian Stretch
- [ ] Raspbian Jessy
- [ ] Other - tell us more:

**Comment by sanjayrd** [comment]
Hi @skyfall1970,

Take a look at the following [code](https://github.com/alexa/avs-device-sdk/blob/master/ApplicationUtilities/Resources/Audio/src/AlertsAudioFactory.cpp) and it's enclosing [folder](https://github.com/alexa/avs-device-sdk/tree/master/ApplicationUtilities/Resources). In here, we read the sounds to play for Alerts and Timers from `.wav` files and play them whenever an alert needs to sound.

Basically, we just read a `.wav` files into `std::istream` and feed this into the MediaPlayer via the following [API](https://github.com/alexa/avs-device-sdk/blob/master/MediaPlayer/include/MediaPlayer/MediaPlayer.h#L84), which handles the playing of an  `std::istream`. Please let us know if you have any further questions.

Thanks,
Sanjay

**Comment by kjkh** [comment]
If there are any further questions, please re-open.
